samples/drivers/mbox/remote/boards/frdm_mcxn947_mcxn947_cpu1.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_MBOX_NXP_IMX_MU=y
+CONFIG_BUILD_OUTPUT_INFO_HEADER=y
```

Comment:
Typically this only needs to be set when we are copying the secondary image into RAM (and therefore need to know the VMA the image should be loaded at). Have you tried building the sample without this Kconfig set?